index.js

```diff
@@ -1 +1,21 @@
-// Your code here
+// refactored from https://www.sitepoint.com/create-jquery-digital-clock-jquery4u/
+
+'use strict';
+
+var DigitalClock = function () {
+  // new instance of Date
+  var currentTime = new Date ( );
+  
+  var timeString = currentTime.toTimeString();
+  var timePieces = timeString.split(" ");
+  var firstPiece = timePieces[0];
+  var clockDiv = $("#clock"); 
+
+  clockDiv.html(firstPiece);
+} 
```

Comment:
Clever use of split to get the current time with less hassle than using `getMinutes` etc.